modules/nf-core/gatk4/filtermutectcalls/environment.yml

```diff
@@ -1,5 +1,9 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
 channels:
   - conda-forge
   - bioconda
 dependencies:
-  - bioconda::gatk4=4.5.0.0
+  # renovate: datasource=conda depName=bioconda/gatk4
+  - bioconda::gatk4=4.6.1.0
+  - bioconda::gcnvkernel=0.9
```

Comment:
I am noticing we are having the gcnvkernel now everywhere. why?

Reply:
I suspect it is so that all the gatk4 modules use the same container, and tying conda up with that

Reply:
Same comment as earlier. When doing the update I was told to have the same container and dependencies for each subtool

Reply:
ok interesting. I missed that development. Thank you for filling me in ❤️